src/agent/onefuzz/src/jitter.rs

```diff
@@ -13,3 +13,9 @@ pub fn jitter(value: Duration) -> Duration {
 pub async fn delay_with_jitter(value: Duration) {
     delay_for(jitter(value)).await
 }
+
+pub async fn delay_with_jitter_size(value: Duration) {
+    let random: u64 = thread_rng().gen_range(0, value.as_secs());
```

Comment:
1. Do we really want to allow a "delay" of 0?

2. It looks like this function is not really a delay with jitter, but more of a uniformly random delay, parametric in an upper bound. I suggest we just rename it to avoid confusion, given that right it looks misleadingly similar to a function where the user-provided value is a _lower_ bound.

Reply:
Sigh, resolve comment doesn't include feedback.

1. Some amount of delay of zero is fine here.  The intent is to reduce the initial heartbeat storm.   
2. renamed per feedback